twine-upload.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Just a nitpick but I usually prefer a space after the shebang marker, for readability.
```suggestion
#! /usr/bin/env bash
```